src/globalStyles.js

```diff
@@ -20,4 +27,20 @@ export const GlobalStyle = createGlobalStyle`
       outline: none;
     }
   }
+
+  .h1, .h2, .h3, h1, h2, h3 {
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
+
+  figure {
+    margin: 0;
+  }
+
+  .h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
+    font-family: inherit;
+    font-weight: 500;
+    line-height: 1.1;
+    color: inherit;
+  }
```

Comment:
Are you sure we need this on all pages?
Isn't it used only on login page?

Reply:
I need these styles in Global or I must add it in every Slide from LoginPage. What would you prefer?

Reply:
Why on every slide? :)
Why can't we add it for the whole login page, don't we have a `LoginPageComponent`?

Reply:
We surely don't want to duplicate this code 5 times (for every slide).
If it's not possible to add these styles for the whole login page, and we have to keep them here in global, please add a comment
```
/**
 * Styles for Login page
 */
... code which is used only for login page ...

Reply:
Sorry, that was my fault. I was wrong. I can do this style working from `LoginPageComponent`